templates/default/snapraid_runner.conf.erb

```diff
@@ -1,14 +1,17 @@
+<%
+path_separator = node['platform'] == 'windows' ? '\\' : '/'
+%>
 [snapraid]
 ; path to the snapraid executable (e.g. /bin/snapraid)
-executable = /usr/local/bin/snapraid
+executable = <%=node['snapraid']['install_directory'] + path_separator%>snapraid<%= node['platform'] == 'windows' ? '.exe' : ''%>
```

Comment:
I would try and push this down into a computed attribute. Easier, and testable in chefspec.